query-engine/prisma/src/tests/type_mappings/mysql_types.rs

```diff
@@ -82,21 +82,21 @@ async fn mysql_types_roundtrip(api: &TestApi) -> TestResult {
             .assert_field_type("string_text_text", ScalarType::String)?
             .assert_field_type("string_text_mediumtext", ScalarType::String)?
             .assert_field_type("string_text_longtext", ScalarType::String)?
-            .assert_field_type("string_binary_binary", ScalarType::String)?
-            .assert_field_type("string_blob_tinyblob", ScalarType::String)?
-            .assert_field_type("string_blob_mediumblob", ScalarType::String)?
-            .assert_field_type("string_blob_blob", ScalarType::String)?
-            .assert_field_type("string_blob_longblob", ScalarType::String)?
+            // .assert_field_type("string_binary_binary", ScalarType::String)?
+            // .assert_field_type("string_blob_tinyblob", ScalarType::String)?
+            // .assert_field_type("string_blob_mediumblob", ScalarType::String)?
+            // .assert_field_type("string_blob_blob", ScalarType::String)?
+            // .assert_field_type("string_blob_longblob", ScalarType::String)?
             .assert_field_enum_type("string_enum", "types_string_enum")?
-            .assert_field_type("string_set", ScalarType::String)?
-            .assert_field_type("spatial_geometry", ScalarType::String)?
-            .assert_field_type("spatial_point", ScalarType::String)?
-            .assert_field_type("spatial_linestring", ScalarType::String)?
-            .assert_field_type("spatial_polygon", ScalarType::String)?
-            .assert_field_type("spatial_multipoint", ScalarType::String)?
-            .assert_field_type("spatial_multilinestring", ScalarType::String)?
-            .assert_field_type("spatial_multipolygon", ScalarType::String)?
-            .assert_field_type("spatial_geometrycollection", ScalarType::String)?
+            // .assert_field_type("string_set", ScalarType::String)?
+            // .assert_field_type("spatial_geometry", ScalarType::String)?
+            // .assert_field_type("spatial_point", ScalarType::String)?
+            // .assert_field_type("spatial_linestring", ScalarType::String)?
+            // .assert_field_type("spatial_polygon", ScalarType::String)?
+            // .assert_field_type("spatial_multipoint", ScalarType::String)?
+            // .assert_field_type("spatial_multilinestring", ScalarType::String)?
+            // .assert_field_type("spatial_multipolygon", ScalarType::String)?
+            // .assert_field_type("spatial_geometrycollection", ScalarType::String)?
```

Comment:
These are commented out here, because they are commented out in the introspected schema.